### PR TITLE
Add suffixes to typos from "ep" to "exe"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21592,6 +21655,7 @@ erturning->returning
 erturns->returns
 ervery->every
 erverything->everything
+ervice->service
 ervices->services
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21659]:
```suggestion
ervices->services
ervices->services, cervices,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -21592,6 +21655,7 @@ erturning->returning
 erturns->returns
 ervery->every
 erverything->everything
+ervice->service
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21658]: Any `crevice` typos out there?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -21666,16 +21739,32 @@ especialy->especially
 especialyl->especially
 especifically->specifically, especially,
 especiially->especially
-espect->expect
+espect->expect, aspect, respect,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:21742]: No `inspect` or `suspect` typos out there?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -21905,6 +22006,8 @@ evovler->evolver
 evovling->evolving
 evrithing->everything
 evry->every
+evrybody->everybody
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22009]: Perhaps add typos of `everyday`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -21974,11 +22098,17 @@ exapnds->expands
 exapnsion->expansion
 exapnsions->expansions
 exat->exact
+exatc->exact
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:22101]: Add typos of `exacta`? Might be useful in the context of probability.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:21658]: I couldn't find any.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:21742]: I couldn't find `inspect` or `suspect`.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**